web/packages/core/src/polyfills.ts

```diff
@@ -189,11 +189,25 @@ async function injectRuffle(
  */
 function initMutationObserver(): void {
     const observer = new MutationObserver(function (mutationsList) {
-        // If any nodes were added, re-run the polyfill to detect any new instances.
-        const nodesAdded = mutationsList.some(
-            (mutation) => mutation.addedNodes.length > 0,
+        // If any embed or object nodes were added, re-run the polyfill to detect any new instances.
+        const elements = ["EMBED", "OBJECT"];
+        const embedOrObjectAdded = mutationsList.some((mutation) =>
+            Array.from(mutation.addedNodes).some(
+                (node) =>
+                    elements.includes(node.nodeName) ||
+                    document
+                        .createTreeWalker(
```

Comment:
This is needed because of https://stackoverflow.com/questions/61314922/mutationobserver-not-picking-up-on-child-nodes